client/src/contexts/CanvasContext.tsx

```diff
@@ -126,7 +128,7 @@ export const CanvasProvider: React.FC<CanvasProviderProps> = ({
         const x = pointer.x;
         const y = pointer.y;
 
-        socket?.volatile.emit("cursor-move", { x, y, color: "#aaf" } as any);
+        socket?.volatile.emit("cursor-move", { x, y, color: getColorFromEmail(decodedToken?.email ?? "") } as any);
```

Comment:
I wonder if it would be worth memorizing the result of the `getColorFromEmail` function with the `useMemo` hook? The `decodedToke?.email` will rarely change